src/CMake/cpackLin.cmake

```diff
@@ -155,7 +155,20 @@ else ()
   SET (CPACK_GENERATOR "TGZ")
 endif()
 
-SET(CPACK_PACKAGE_FILE_NAME "${CPACK_PACKAGE_NAME}_${XRT_VERSION_RELEASE}.${CPACK_PACKAGE_VERSION_MAJOR}.${CPACK_PACKAGE_VERSION_MINOR}.${CPACK_PACKAGE_VERSION_PATCH}_${CPACK_REL_VER}-${CPACK_ARCH}")
+# On Amazon Linux CPACK_REL_VER is just '2' and it is hard to
+# distinguish it as AL package, so adding CPACK_FLOVOR (eg: amzn) 
+# to package name
+if (${LINUX_FLAVOR} MATCHES "^Amazon")
+  execute_process(
+    COMMAND awk -F= "$1==\"ID\" {print $2}" /etc/os-release
+    COMMAND tr -d "\""
```

Comment:
Since you are using the AWK programming language, you should not need `tr` too, for example by using the `sub()` function.
But this is not fundamental.

Reply:
Hi @keryell, in both cases o/p is different -> lsb_release --id --short gives **Amazon** as o/p but all the standard packages uses **amzn** in their versions, so I used o/p from file '/etc/os-release'. 

eg: yum info git
Name        : git
Arch        : x86_64
Version     : 2.32.0
Release     : **1.amzn2.0.1**
Size        : 365 k
Repo        : installed
From repo   : **amzn2-core**
Summary     : Fast Version Control System
URL         : https://git-scm.com/
License     : GPLv2

But as it is just a part of package name we can give any name we want, o/p from lsb_release --id should also do

Reply:
We are trying to add OS names to other distribution(ubuntu,centos,rhel) packages as well, while handling them we can decide whether to use info from lsb_release or /etc/os-release file

Reply:
But what I do not understand is what is not already in https://cmake.org/cmake/help/latest/command/cmake_host_system_information.html#command:cmake_host_system_information

Reply:
That will add additional requirement of cmake 3.22+. 
We can either use existing approach or we can even use lsb_release --id --short in the package name.

Reply:
Do you mean the information you want is not available in an older version of CMake, like 3.10?

Reply:
Yes @keryell  . Distribution information is added in 3.22.

Reply:
Ah, too bad!

Reply:
Yeah some of the centos and rhel OSes we support still doesn't have this support, they use cmake of 3.17 version

Reply:
@rbramand-xilinx , So, whatever you have implemented is good then. Merging the change..